docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/extension-compatibility.md

```diff
@@ -28,45 +28,15 @@ TFS 2017 Update 1 is technically supported with version 2.0.199 of the extension
 
 Any version older than TFS 2015 Update 2 is not supported by any extension version.
 
-You may have success manually installing the task by following [these instructions](manually-install-the-build-task.md).
-
 ### Build information compatibility {#build-information-compatibility}
 
-When passing [build information](docs/packaging-applications/build-servers/build-information/index.md) to Octopus from TFS, you may encounter issues when trying to use the build link generated by the Octopus TFS extension.
+When passing [build information](docs/packaging-applications/build-servers/build-information/index.md) to Octopus from TFS, you may encounter issues when trying to use the build link generated by the TFS extension.
```

Comment:
TFS => Azure DevOps ?

Reply:
@slewis74 I am actually considering completely remove "Build information compatibility" section.
This section seems like we are documenting bugs in the MS product, thoughts?

Reply:
@markryd you have added this section, can you provide more context on this?

Reply:
> @harrisonmeister you have added this section, can you provide more context on this?

The context behind it was mentioned by Support that it affected at least two customers. One was this [ticket](https://help.octopus.com/t/package-build-information-tfs-2017-update-3/26028). I can't say if it's still an issue. Since you're doing a revamp, I'm happy to defer to your expertise on this subject if it would continue to be an issue or not (or better: whether we still want to point out the differences in our docs)